ci/lava/tests/test-provision-mbl.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Tests for MBL provisioning."""
+
+import os
+import time
+from pathlib import Path
+
+
+class TestProvisionMbl:
+    """Class to encapsulate the testing of mbl-cli provisioning of a DUT."""
+
+    dut_addr = ""
+    certificate = ""
+
+    def test_setup(self, dut_addr, execute_helper):
+        """Setup enviroment for tests execution."""
+        # Save the dut_addr in the class
+        TestProvisionMbl.dut_addr = dut_addr
+
+        # Work out a unique certificate id - the first part of the path is the
+        # lava job number.
+        current_dir = Path().cwd()
+        TestProvisionMbl.certificate = "".join(current_dir.parts[1])
+
+        # Create a working directory for the following tests
+        directory = "/tmp/update-resources"
+        if not os.path.exists(directory):
+            os.mkdir(directory)
+        os.chdir(directory)
+
+    def test_manifest_ini(self, execute_helper):
```

Comment:
This name is confusing, why isn't this called `test_manifest_init`?

Reply:
Fixed: 4be8f3e7d0a7fe405081c27aa914551079d59977